gmail/src/models/error_message.ts

```diff
@@ -5,9 +5,11 @@ import { _t } from "../services/translation";
  */
 
 const _ERROR_CODE_MESSAGES: Record<string, string> = {
+    odoo: null, // Message is contained in the additional information
```

Comment:
Mmmh ?

Reply:
To not show the "title" on the error card view and only show the error returned by Odoo